api/service/CampaignService.py

```diff
@@ -1,12 +1,24 @@
 from flask import jsonify
+from api.helper.db import db
+from api.model.Campaign import CampaignModel
 
 
 class CampaignService:
 
     def getCampaigns(self):
-        result = {"status": "success"}
-        return jsonify(result)
+        campaigns = CampaignModel.query.all()
+        results = [
+            {
+                "name": campaign.name,
+                "active": campaign.active,
+            } for campaign in campaigns]
+
+        return {"status": "success", "data": results}
```

Comment:
we should do some error handling here. Probably add an exception to catch and log errors in case of failure

Reply:
What we have now is just to test the application/db docker container read/write. Proper error handling will be used with implementation